aries_cloudagent/protocols/out_of_band/v1_0/manager.py

```diff
@@ -637,7 +689,7 @@ async def _process_pres_request_v1(
                     holder=self._session.inject(IndyHolder),
                 )
             except ValueError as err:
-                self._logger.warning(f"{err}")
+                LOGGER.warning(f"{err}")
```

Comment:
Maybe add some context so it's clear where the logged error is coming from

Reply:
Actually I think you can use `LOGGER.exception("...")` and avoid capturing `err` manually.